src/themes/clean/templates/elements/journal/issue_list_by_decade.html

```diff
@@ -6,7 +6,9 @@
 <ul class="list-inline">
     {% for decade, issues in issues_by_decade.items %}
         <li class="list-inline-item">
-            <a href="#decade-{{ decade }}">{{ decade }}</a>
+            <a href="#decade-{{ decade }}">
+                {{ decade }}
+            </a>
```

Comment:
Try to avoid formatting changes that are not related to the issues we are addressing